templates/data/lakehouse/index.html

```diff
@@ -118,54 +118,66 @@ <h2>Data lakehouse architecture</h2>
                     </div>
                 </div>
 
-                <div class="row">
-                    <div class="col-3 col-start-large-4">
-                        <h3 class="p-heading--5">Ingestion</h3>
-                    </div>
+                <div class="p-section--shallow">
+                    <div class="row">
+                        <div class="col-3 col-start-large-4">
+                            <h3 class="p-heading--5">Ingestion</h3>
+                        </div>
 
-                    <div class="col-6">
-                        <p>
-                            This initial layer consists of acquiring data from various sources, processing it, and converting it into a format suitable for storage and analysis within the lakehouse.
-                        </p>
+                        <div class="col-6">
+                            <p>
+                                This initial layer consists of acquiring data from various sources, processing it, and converting it into a format suitable for storage and analysis within the lakehouse.
+                            </p>
+                        </div>
                     </div>
                 </div>
-                <div class="row">
-                    <div class="col-9 col-start-large-4">
-                        <hr class="is-muted" />
+                <div class="p-section--shallow">
+                    <div class="row">
+                        <div class="col-9 col-start-large-4">
+                            <hr class="is-muted" />
```

Comment:
```suggestion
                            <hr class="p-rule--muted" />
```
We should update the muted class to the latest convention

Reply:
That's straight from Vanilla pattern's generated HTML for tiered list (also the missing shallows you mentioned earlier). I have a [PR](https://github.com/canonical/vanilla-framework/pull/5503) up there to add an image feature to tiered list, so this entire section is going to be replaced by a pattern macro.